app/models/loan.rb

```diff
@@ -237,4 +237,19 @@ def change_in_principal(start_date: nil, end_date: nil)
     raise Accounting::TransactionDataMissingError if changes.any?(&:blank?)
     changes.sum
   end
+
+  def default_accounting_customer_for_transaction(transaction)
```

Comment:
I believe we discussed changing this in our last meeting, so this PR may be a WIP but I'll keep reviewing it for now anyway.